src/renderers/shaders/ShaderChunk/lights_lambert_vertex.glsl

```diff
@@ -101,28 +65,33 @@ vec3 normal = normalize( transformedNormal );
 
 #endif
 
-#if MAX_HEMI_LIGHTS > 0
+	{
+		// dotNL is always one, and diffuseColor is vec3(1.0), thus the result is equivalent to summing indirectDiffuse lights
```

Comment:
This sentence as an explanation does not make sense. I'll fix comments later.

A hemi-light is a model for irradiance, and since we are assuming a Lambertian surface, 

    radaince = brdf * irradiance.

I will need to make a mental note to revisit the factor-of-pi issue in this calculation.